library/src/interactor_impl.cxx

```diff
@@ -540,8 +540,6 @@ class interactor_impl::internals
   vtkNew<vtkRenderWindowInteractor> VTKInteractor;
   vtkNew<vtkF3DInteractorStyle> Style;
   vtkSmartPointer<vtkF3DInteractorEventRecorder> Recorder;
-  int WindowSize[2] = { -1, -1 };
```

Comment:
How did you notice that? Why is cppcheck or clang tidy failed to detect it?

Reply:
local cppcheck detected it.